### PR TITLE
Add headers to KNOWN_ISSUES

### PR DESCRIPTION
**Pull request by mojito317**
## PR summary

Since the `headers` parameters in the request bodies must be used as `headers_` in Python to prevent collision with the request headers (that are `headers` in the Python-wrapped request), it is good to have a section about this different behavior in the KNOWN_ISSUES.

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [x] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: KNOWN_ISSUES.md [hunk 1]
```diff
@@ -47,3 +47,35 @@ consult the Cloudant documentation for further information.
 <!-- Template substitution for language specific content -->
 <!-- ## SPLIT MARKER ## -->
 ## Cloudant SDK for Python
+
+### Bodies containing the `headers` parameter
```
Comment (bessbd) [inline KNOWN_ISSUES.md:51]: Nit

```suggestion
### Request bodies containing the `headers` parameter
```

File: KNOWN_ISSUES.md [hunk 2]
```diff
@@ -47,3 +47,35 @@ consult the Cloudant documentation for further information.
...
+
+The `headers` dict is reserved for the API request headers.
+In order to avoid collision between the request headers and the `headers` parameter
+that is contained by the request body, use the `headers_` as
+the request body `headers` parameter.
+
+E.g. for applying the `headers` parameter of your replicate
```
Comment (bessbd) [inline KNOWN_ISSUES.md:58]: Nit
```suggestion
E.g. to set the `headers` parameter of your replication
```

**Review by bessbd — APPROVED** [review]

File: KNOWN_ISSUES.md (hunk 1 above)
Comment (mojito317) [reply KNOWN_ISSUES.md]: Fixed in d1ac0de.

File: KNOWN_ISSUES.md (hunk 2 above)
Comment (mojito317) [reply KNOWN_ISSUES.md]: Fixed in d1ac0de.

**Review by emlaver — APPROVED** [review]

**Comment by mojito317** [comment]
Thanks for the reviews @bessbd and @emlaver! I'm about to merge this!

**Pull request merged**